FFXIVClientStructs/FFXIV/Client/Game/Character/DrawDataContainer.cs

```diff
@@ -6,39 +8,55 @@ public unsafe partial struct DrawDataContainer {
-    [FieldOffset(0x0DC)] public byte OffHandFlags2;
-
-    [FieldOffset(0x0E0)] public WeaponModelId UnkE0Model;
-    [FieldOffset(0x0F0)] public DrawObjectData UnkF0;
-    [FieldOffset(0x142)] public ushort Unk144Flags1;
-    [FieldOffset(0x144)] public byte Unk144Flags2;
-
-    [FieldOffset(0x148)] public EquipmentModelId Head;
-    [FieldOffset(0x14C)] public EquipmentModelId Top;
-    [FieldOffset(0x150)] public EquipmentModelId Arms;
-    [FieldOffset(0x154)] public EquipmentModelId Legs;
-    [FieldOffset(0x158)] public EquipmentModelId Feet;
-    [FieldOffset(0x15C)] public EquipmentModelId Ear;
-    [FieldOffset(0x160)] public EquipmentModelId Neck;
-    [FieldOffset(0x164)] public EquipmentModelId Wrist;
-    [FieldOffset(0x168)] public EquipmentModelId RFinger;
-    [FieldOffset(0x16C)] public EquipmentModelId LFinger;
-
-    [FieldOffset(0x188)] public CustomizeData CustomizeData;
-
-    [FieldOffset(0x1A2)] public uint Unk18A;
-    [FieldOffset(0x1A6)] public byte Flags1;
-    [FieldOffset(0x1A7)] public byte Flags2;
+    [Obsolete("Use Weapon(WeaponSlot.OffHand).ModelId", true)]
+    [FieldOffset(0x080)] public WeaponModelId OffHandModel;
+    [Obsolete("Use Weapon(WeaponSlot.OffHand).State", true)]
+    [FieldOffset(0x0DC)] public byte OffHandState;
+    [Obsolete("Use Weapon(WeaponSlot.OffHand).Flags1", true)]
+    [FieldOffset(0x0E2)] public ushort OffHandFlags1;
+    [Obsolete("Use Weapon(WeaponSlot.OffHand).Flags2", true)]
+    [FieldOffset(0x0E4)] public byte OffHandFlags2;
+
+    [Obsolete("Use Weapon(WeaponSlot.Unk).ModelId", true)]
+    [FieldOffset(0x0F0)] public WeaponModelId UnkE0Model;
+    [Obsolete("Use Weapon(WeaponSlot.Unk).Flags1", true)]
+    [FieldOffset(0x152)] public ushort Unk144Flags1;
+    [Obsolete("Use Weapon(WeaponSlot.Unk).Flags2", true)]
+    [FieldOffset(0x154)] public byte Unk144Flags2;
+
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x00)] public EquipmentModelId Head;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x04)] public EquipmentModelId Top;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x08)] public EquipmentModelId Arms;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x0C)] public EquipmentModelId Legs;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x10)] public EquipmentModelId Feet;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x14)] public EquipmentModelId Ear;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x18)] public EquipmentModelId Neck;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x1C)] public EquipmentModelId Wrist;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x20)] public EquipmentModelId RFinger;
+    [FieldOffset(0x010 + 3 * DrawObjectData.Size + 0x24)] public EquipmentModelId LFinger;
+
+    [FieldOffset(0x200)] public CustomizeData CustomizeData;
+
+    [FieldOffset(0x1BA)] public uint Unk18A;
+    [FieldOffset(0x1BE)] public byte Flags1;
+    [FieldOffset(0x1BF)] public byte Flags2;
```

Comment:
CustomizeData, Unk18A, Flags1, Flags2 was already updated for 6.5 change these back

Reply:
![img](https://screenshots.wildwolf.dev/ReClass.NET_b435589a37.png) As you can see in this picture, the old offsets before this pr is the correct ones